### ISSUE TITLE
Show branch names above diff

### ISSUE THREAD

**Issue by greggman** [sublimehq/sublime_merge#548]
**Problem description**

When I select 2 branches I can't tell which is which

**Preferred solution**

Put branch names at the top of each side of the diff columns, sticky so as I scroll down the diff I can tell at a glance which column is which. If there isn't enough room for both columns then add some header at the top that shows by color which changes are from which branch.

**Screenshots**

<img width="1753" alt="Screen Shot 2019-07-25 at 19 36 35" src="https://user-images.githubusercontent.com/234804/61868444-5c0be180-af14-11e9-8cd7-fef084d3fc2e.png">

**Comment by srbs** [comment]
I agree it isn't clear, but here is an explanation of what you _can_ see:

The top section contains key/value pairs for various pieces of information. Every value pertains to the right hand side (or the 'diff-to'/additive portion) _except_ the `Diffing From` row. Knowing that, you can determine that the listed branch, `ugh`, is on the right & thus the `master` branch is on the left. You can confirm this behavior by verifying the `Diffing From` commit hash is the same commit that `master` points to.

From what I can determine, the top selected commit is always on the right & the bottom is always on the left. You can select more than two commits and the top & bottom most commits are the ones on the left/right.

**Comment by greggman** [comment]
> From what I can determine, the top selected commit is always on the right & the bottom is always on the left. You can select more than two commits and the top & bottom most commits are the ones on the left/right.

In real git I can compare A to B and B to A so to do that same you need to be able to select which is left and which is right, not have it based on position on in the commit history
